fastlane/actions/get_version_code.rb

```diff
@@ -0,0 +1,45 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      FL_CURRENT_VERSION_CODE = :CURRENT_VERSION_CODE
+    end
+
+    class GetVersionCodeAction < Action
+      def self.run(params)
+        File.open 'app/version.properties' do |file|
+          setting = file.find { |line| line =~ /([0-9])+/ }
+          version_code = setting[/([0-9])+/]
+        end
+      end
```

Comment:
The only requirement is that the `app/version.properties` file exist and has -some- preexisting version number, and we could adjust it so that it handles the case where that file does not exist.

This action simply retrieves that value, another action takes care of incrementing it.

Reply:
The version code should be ephemeral. Lets get it from Appcenter because we are never going to update the version code in this repo.